File: ALittle/Module/ALittleIDE/Script/UIEdit/IDEEditCore/IDESetting/TextRadioButtonS.lua
-- ALittle Generate Lua
module("ALittleIDE", package.seeall)

local ___rawset = rawset
local ___pairs = pairs
local ___ipairs = ipairs


assert(ALittleIDE.TextCheckButtonS, " extends class:ALittleIDE.TextCheckButtonS is nil")
TextRadioButtonS = Lua.Class(ALittleIDE.TextCheckButtonS, "ALittleIDE.TextRadioButtonS")

function TextRadioButtonS:Ctor(user_info, tab_child, tree_logic)
	___rawset(self, "_layer_name", "ide_setting_textradiobutton")
end

function TextRadioButtonS:LoadNatureBase()
	TextCheckButtonS.LoadNatureBase(self)
end

File: ALittleIDE/Module/ALittleIDE/JSScript/IDE/UIEdit/Setting/TextRadioButtonS.js
{
if (typeof ALittleIDE === "undefined") window.ALittleIDE = {};


if (ALittleIDE.TextCheckButtonS === undefined) throw new Error(" extends class:ALittleIDE.TextCheckButtonS is undefined");
ALittleIDE.TextRadioButtonS = JavaScript.Class(ALittleIDE.TextCheckButtonS, {
	Ctor : function(user_info, tab_child, tree_logic) {
		this._layer_name = "ide_setting_textradiobutton";
	},
	LoadNatureBase : function() {
		ALittleIDE.TextCheckButtonS.LoadNatureBase.call(this);
	},
}, "ALittleIDE.TextRadioButtonS");

}